Installing-and-Configuring/OS/index.md

```diff
@@ -77,9 +83,11 @@ This feature may be turned off during configuration of heads.  The following wil
   CONFIG_TPM_NO_LUKS_DISK_UNLOCK=y
 
 
-### Using only a LUKS passphrase
+### Using the LUKS Disk Recovery Key / Using only a LUKS passphrase
+
+In this model heads is not involved in processes related to the decryption of data on disk. The OS manages keys and access to the LUKS containers by asking the user for a passphrase on boot.  
 
-In this model heads is not responsible for any of the decryption of data on disk.  The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase.  The correct passphrases allows boot to finish.
+In the Disk Recovery Key scenario, Heads doesn't attempt to release the Disk Unlock Key. It simply relies on OS mechanisms to prompt the user for the Disk Recovery Key passphrase at standard OS prompt.
```

Comment:
In the Disk Recovery Key scenario, Heads doesn't attempt to release the Disk Unlock Key. It simply relies on OS mechanisms to directly prompt the user for the Disk Recovery Key passphrase at standard OS prompt. In the Disk Unlock Key scenario, that [key is provided to the OS in the form of a transitional crypttab file, fed to the operating system inside of the initrd file prior of the Operating system booting](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-insert-key#L75-L80). Consequently, the Operating system tries this key which match required decryption key for slot 1 and the OS continues to boot without prompting the user for the Disk Recovery Key passphrase.